bin/gobblin.sh

```diff
@@ -417,7 +427,15 @@ function start() {
             CLASS_N_ARGS=''
             if [[ "$GOBBLIN_MODE" = "$STANDALONE_MODE" ]]; then
                 CLASS_N_ARGS="$STANDALONE_CLASS $GOBBLIN_CONF/application.conf"
+                ADDITIONAL_ARGS="-Dgobblin.logs.dir=${GOBBLIN_LOGS}"
+
+                if [ -z "$GOBBLIN_WORK_DIR" ] && [ "$CHECK_ENV_VARS" == true ]; then
```

Comment:
I originally removed the ENV variables deliberately to make one way of controlling everything. but i can see ENV variable can be additional way of configuring too. but should not be the only way. may be we can merge it or make it overridable by user config for GOBBLIN_JOB_CONFIG_DIR and GOBBLIN_WORK_DIR.